to-do-list/src/Components/Context.js

```diff
@@ -0,0 +1,3 @@
+import React,{useState} from 'react';

```

Comment:
Move this file in services folder as explain in App.js file

Remove un-used variable.,
Import `createContext` as module